warp/_src/codegen.py

```
@@ -2291,6 +2291,28 @@ def get_unroll_range(adj, loop):
 
             ok_to_unroll = True
 
+            contains_static = False
+            for node in ast.walk(loop):
+                if not isinstance(node, ast.Call):
+                    continue
+                try:
+                    func, _ = adj.resolve_static_expression(node.func, eval_types=False)
+                except Exception:
+                    continue
```

Ruff (0.14.4)

`try`-`except`-`continue` detected, consider logging the exception

(S112)